mandatory/3-function-output.js

```diff
@@ -1,18 +1,21 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
-  return Math.random() * 10;
+  return Math.random() * 10; //The Math. random() function returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1) with approximately uniform distribution over that range
```

Comment:
This is a great copy/paste from MDN Web Docs :) what's a floating-point, pseudo-random number?

Reply:
you are right , as it says we meant to google it , I just copy paste what I found and have no idea what that means :)) I eaven searched the meaning of it now and it got harder to understand ,I believe that Math.random() will return a number between 0 and 1 , I think it could include 0 but not the 1

Reply:
Nice! Any idea what kind of number? Positive? Negative? Decimal? Something else?